packages/shopify-codemod/transforms/utils.js

```diff
@@ -14,6 +14,24 @@ export function findLastMember(node) {
   return node;
 }
 
+export function forEachAssignment(assignment, cb) {
```

Comment:
I'm not a huge fan of the helper in its current form. I'd rather see each assignment expression captured and returned as an array, making it call the callback as well feels awkward and out of place with our other helpers.